Userland/Applications/PDFViewer/PDFViewerWidget.cpp

```diff
@@ -103,11 +104,18 @@ void PDFViewerWidget::open_file(const String& path)
 {
     window()->set_title(String::formatted("{} - PDF Viewer", path));
     auto file_result = Core::File::open(path, Core::OpenMode::ReadOnly);
-    VERIFY(!file_result.is_error());
+    if (file_result.is_error()) {
+        GUI::MessageBox::show_error(nullptr, String::formatted("Couldn't open file: {}\n", path));
+        return;
+    }
+
     m_buffer = file_result.value()->read_all();
     auto document = PDF::Document::create(m_buffer);
-    // FIXME: Show error dialog if the Document is invalid
-    VERIFY(document);
+    if (!document) {
+        GUI::MessageBox::show_error(nullptr, String::formatted("Couldn't load PDF: {}\n", path));
```

Comment:
Is the newline intentional? seems a bit weird to have it in a message box

Reply:
Uhh idk, I think I just copied it from some other code. I'll remove it though